### ISSUE TITLE
Mailchimp API response messages language

### ISSUE THREAD

**Issue by dedric81** [rainlab/mailchimp-plugin#34]
Anyone know if is possible translate error messages from English (default) to another language? 
I need to traslate errors in Italian.
thx

**Comment by bennothommo** [comment]
@dedric81 Which error messages are you referring to? If you mean the messages in the component, you would need to override the `components/signup/results.htm` partial within your theme. You may follow [these instructions](https://octobercms.com/docs/cms/components#overriding-partials) in order to do this.

**Comment by dedric81** [comment]
Sorry, I mean messagges of mailchimp api, not only errors. 

Exemple: "Sorry there was an error: 400: myemail@email.com is already a list member. Use PUT to insert or update list members._

https://mailchimp.com/developer/guides/error-glossary/

**Comment by bennothommo** [comment]
@dedric81 I don't believe you can change the error language from Mailchimp directly. Translating those would require the plugin to intercept those errors and then provide a translatable message, as opposed to simply passing through the error from Mailchimp.
